shared/settings/email/container.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+import React, {Component} from 'react'
+import UpdateEmail from './index'
+import {connect} from 'react-redux'
+import {navigateUp} from '../../actions/router'
+import {onSubmitNewEmail, onChangeNewEmail} from '../../actions/settings'
+
+import type {Props} from './index'
+import type {TypedState} from '../../constants/reducer'
+
+class UserEmailContainer extends Component<void, Props, void> {
```

Comment:
🐑 You can use the Routable HOC

It would look like:

```
const UserEmailContainer = Routable(() => ({componentAtTop: {title: 'Change Email'}}), UpdateEmail)
```
In util/routable